src/theory/builtin/theory_builtin_rewriter.cpp

```diff
@@ -70,6 +77,183 @@ Node TheoryBuiltinRewriter::blastChain(TNode in) {
   }
 }
 
+
+// recursion depth is bounded by the number of arguments in the bound variable list bvl
```

Comment:
Move this comment to header file.

Reply:
We need to do this uniformly, I was following the style of Aina's comments on my other branch regarding recursion depth which was added to the cpp:
https://github.com/CVC4/CVC4/commit/d7dadde871ae4775748695b0b7f9deee49576c0a#diff-c473e454b74828446a5f7030f7840815R590
I think it is more appropriate to comment on technical issues like recursion depth in the cpp.

Reply:
My preference would be to have comments about the function as a whole in the header and technical comments about individual lines of code in the cpp.  So, if this comment were moved to the line where the recursive call occurs, that would be fine.  But because the function has "Rec" in it's name, it seems more appropriate for it to be a function-level comment, and then it should go in the header.  We can discuss this further in tomorrow's meeting.